docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,97 @@
+# Model Optimizer Python API {#openvino_docs_MO_DG_Python_API}
+
+@sphinxdirective
+
+Model Optimizer (MO) has a Python API for model conversion, which is represented by the ``convert_model()`` method in the openvino.tools.mo namespace.
+
+  ``convert_model()`` has all the functionality available from the command-line tool, plus the ability to pass Python model objects, such as a Pytorch model or TensorFlow Keras model directly, without saving them into files and without leaving the training environment (Jupyter Notebook or training scripts). In addition to input models consumed directly from Python, ``convert_model`` can take OpenVINO extension objects constructed directly in Python for easier conversion of operations that are not supported in OpenVINO.
+  ``convert_model()`` returns an openvino.runtime.Model object which can be compiled and inferred or serialized to IR.
+
+.. code-block:: python
+
+   import torchvision
+   
+   model = torchvision.models.resnet50(pretrained=True)
+   ov_model = convert_model(model)
+
+The following types are supported as an input model for ``convert_model()``:
+
+* PyTorch - ``torch.nn.Module``, ``torch.jit.ScriptModule``, ``torch.jit.ScriptFunction``. Refer to the :doc:`Converting a PyTorch Model<openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch>` article for more details.
+* TensorFlow/ TensorFlow2 / Keras - ``tf.keras.Model``, ``tf.keras.layers.Layer``, ``tf.compat.v1.Graph``, ``tf.compat.v1.GraphDef``, ``tf.Module``, ``tf.function``, ``tf.compat.v1.session``, ``tf.train.checkpoint``. Refer to the :doc:`Converting a TensorFlow Model<openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow>` article for more details.
+
+``convert_model()`` accepts all parameters available in the MO command-line tool. Parameters can be specified by Python classes or string analogs, similar to the command-line tool.
+
+Example 1:
+
+.. code-block:: python
+
+   from openvino.runtime import PartialShape, Layout
+   
+   ov_model = convert_model(model, input_shape=PartialShape([1,3,100,100]), mean_values=[127, 127, 127], layout=Layout("NCHW"))
+
+Example 2:
```

Comment:
Should not we tell what this example demonstrating? The same comment for other examples.

Reply:
Added comments for all examples.